### ISSUE TITLE
`python3Packages.tensorflow-bin` build failure on x86_64-linux as of `e29253ad`

### ISSUE THREAD

**Issue by samuela** [NixOS/nixpkgs#167442]
## Issue description
Build of `python3Packages.tensorflow-bin` failed on x86_64-linux as of e29253ad9b847873fa8c5d736a45951d67623eef.

[build log](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2099438974)

Maintainer cc: @jyp @abbradar @cdepillabout
Other cc: n/a

### Steps to reproduce
1. Checkout nixpkgs at commit e29253ad9b847873fa8c5d736a45951d67623eef
2. Run `nix-build -A python3Packages.tensorflow-bin`

## Technical details
```
- system: `"x86_64-linux"`
 - host os: `Linux 5.13.0-1017-azure, Ubuntu, 20.04.4 LTS (Focal Fossa), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.7.0`
 - nixpkgs: `/nix/store/ywip1di109ygrclx99m3gi6hgqs61jcc-source`
```

## Misc.
This issue was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2099438974) that created this issue.
- Internal tag: 4070893726358758738

**Comment by samuela** [comment]
Looks like this is an issue with the tensorflow-estimator version:
```
ERROR: Could not find a version that satisfies the requirement tensorflow-estimator<2.8,~=2.7.0rc0 (from tensorflow-cpu) (from versions: none)
[397](https://github.com/samuela/nixpkgs-upkeep/runs/5842880089?check_suite_focus=true#step:11:397)
ERROR: No matching distribution found for tensorflow-estimator<2.8,~=2.7.0rc0
[398](https://github.com/samuela/nixpkgs-upkeep/runs/5842880089?check_suite_focus=true#step:11:398)
```

**Comment by samuela** [comment]
The best course of action to fix this is to upgrade tensorflow-bin to 2.8.0 (currently 2.7). However, there are no macOS wheels for 2.8.0 as far as I can tell (https://github.com/tensorflow/tensorflow/issues/55581).

**Comment by samuela** [comment]
If we don't hear back from upstream TF, I propose that we just drop Darwin support for the time being. Users can always use the default source build which does have Darwin support anyways.
